composer.json

```diff
@@ -44,13 +45,15 @@
         "knplabs/knp-menu-bundle": "^2.1",
         "liip/imagine-bundle": "^1.4",
         "predis/predis": "^1.1",
-        "sensio/framework-extra-bundle": "~3.0",
+        "sensio/distribution-bundle": "^5.0",
+        "sensio/framework-extra-bundle": "^3.0.2",
         "snc/redis-bundle": "~2.0",
         "stof/doctrine-extensions-bundle": "~1.2",
         "symfony/assetic-bundle": "~2.3",
         "symfony/monolog-bundle": "~2.4|~3.0",
-        "symfony/symfony": "^2.8",
-        "symfony/swiftmailer-bundle": "^2.3",
+        "symfony/polyfill-apcu": "^1.0",
+        "symfony/swiftmailer-bundle": "^2.3.10",
+        "symfony/symfony": "~3.0",
```

Comment:
We have to keep Symfony 2.8 compatibility here: `"symfony/symfony": "~2.8|~3.0",`